### ISSUE TITLE
[Accessibility]ARIA role should be appropriate for the element, Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them

### ISSUE THREAD

**Issue by wanghe2015** [MicrosoftDocs/msteams-docs#10651]
### Steps to reproduce

When we go to teams home tab. We found 
<img width="1441" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/13779512/c4948a1c-f3b6-46dc-8d9d-58e7a70ed0b2">

### Expected behavior

1. ARIA role should be appropriate for the element
Role attribute should have an appropriate for the element.
2. Ensure elements marked as presentational are consistently ignored.

### Actual behavior

1. ARIA role should be appropriate for the element
Role attribute should have an appropriate for the element.
2. Ensure elements marked as presentational are consistently ignored.
